xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1473,5 +1477,56 @@ test_build:
         </listitem>
       </itemizedlist>
     </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases.config_url_and_placeholder_variables">
+      <title>Using a Custom Configuration File URL in Combination with Placeholder Variables</title>
+
+      <para>It may happen that you have multiple repositories following the same
+      set of steps, and you would rather have one copy of the configuration
+      file stored in a single place and applied to multiple workflows. The
+      combination of placeholder variables with the configuration file url setting
+      lets you do just that.</para>
+
+      <para>Let's say you have the following configuration file that works with
+      your SCM repository called <emphasis>gnome-shell</emphasis></para>
+
+      <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: "test-project"
+        source_package: "gnome-shell"
+        target_project: "test-target-project"
+  filters:
+    event: pull_request</screen>
+
+      <para>If you replace <emphasis>gnome-shell</emphasis> with
+      <emphasis>%{SCM_REPOSITORY_NAME}</emphasis> like so:</para>
+
+      <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: "test-project"
+        source_package: "%{SCM_REPOSITORY_NAME}"
+        target_project: "test-target-project"
+  filters:
+    event: pull_request</screen>
+
+      <para>It will perform just as well as it did before, however now
+      this configuration can be applied to any other OBS package in the
+      <emphasis>test-project</emphasis> and SCM repository combination assuming
+      they have the same name.</para>
+
+      <para>From here the only thing left to do would be to host this file
+      somewhere where OBS can access it, creating a workflow token and a
+      corresponding webhook (following the <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication">setup instructions</link>)
```

Comment:
```suggestion
      corresponding webhooks (following the <link linkend="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication">setup instructions</link>)
```

Reply:
I would set webhook to plural, so its a bit more clear that each repository needs to setup them, and workflow token is singular since you could use the same for all of them